src/main/java/com/mineboundteam/minebound/MineBound.java

```diff
@@ -28,6 +27,13 @@ public class MineBound {
             return new ItemStack(ItemRegistry.MYRI_MANASAC.get());
         }
     };
+    public static final CreativeModeTab SPELLS_TAB = new CreativeModeTab(MOD_ID) {
+        @OnlyIn(Dist.CLIENT)
+        @Override
+        public @NotNull ItemStack makeIcon() {
+            return new ItemStack(ItemRegistry.TELEKINETIC_OFFENSIVE_1.get());
+        }
+    };
```

Comment:
Is there some way we can rename the tabs? They're currently both named `MineBound`.

Reply:
Yeah, I will; I'm going to hardcode the order, more or less